lib/improver/utilities/cube_checker.py

```diff
@@ -188,7 +188,7 @@ def find_percentile_coordinate(cube):
     if not isinstance(cube, iris.cube.Cube):
         msg = ('Expecting data to be an instance of '
                'iris.cube.Cube but is'
-               ' {0:s}.'.format(type(cube)))
+               ' {0:s}.'.format(str(type(cube))))
```

Comment:
Another string format thing

Reply:
I've removed the `:s` format specifier and removed enforcing `type(cube)` to be a string.